sjb/config/test_cases/test_branch_origin_service_catalog.yml

```diff
@@ -23,6 +25,12 @@ extensions:
       timeout: 1800
       script: |-
         OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/local/bin/linux/amd64 hack/env make build
+    - type: "script"
+      title: "report commit for service catalog"
+      repository: "service-catalog"
+      timeout: 1800
+      script: |-
+        echo Service Catalog commit is $(git log -1 --pretty=%h)
```

Comment:
As discussed out of band, I think for our sanity we should make this git log -2.

Reply:
Yes, updated for -2 and also did the changed for 3.10